sources/Context.h

```diff
@@ -123,6 +123,13 @@ class Context {
    */
   void walkNodesMain();
 
+  /**
+   * @brief Filter models
+   *
+   * Remove models definitions which macro connections are not all performed
```

Comment:
Remove dynamic automaton with partial connectivity

Reply:
OK for "with partial connectivity"